libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFLocalhostSubstitutionCache.m

```diff
@@ -34,13 +35,16 @@
 
 @implementation SFLocalhostSubstitutionCache
```

Comment:
Are we still using SFLocalhostSubstitutionCache ??

Reply:
Is this something we can remove Wolf?

Reply:
I'm not sure yet. Looking through git history.